brianlong/www.validators.app
# 182036856 vote accounts is active

#### What's this PR do?
- add is_active flag for vote_accounts and set it in VoteAccountDetailsService

#### How should this be manually tested?
- run tests
- run migrations
- run gather_vote_account_details_daemon.rb
- check if there are accounts with is_active flag set to false

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182036856)
- https://www.pivotaltracker.com/story/show/182072088

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
